crates/ty_python_semantic/src/types/diagnostic.rs

```diff
@@ -138,7 +174,22 @@ declare_lint! {
 }
 
 declare_lint! {
-    /// TODO #14889
+    /// ## What it does
+    /// Checks for class definitions with multiple metaclasses.
```

Comment:
this isn't accurate -- two classes can be used in multiple inheritance together fine even if they have different metaclasses, as long as one metaclass is a subclass of the other:

```pycon
>>> class M1(type): ...
... 
>>> class M2(M1): ...
... 
>>> class Foo(metaclass=M1): ...
... 
>>> class Bar(metaclass=M2): ...
... 
>>> class Baz(Foo, Bar): ...
...
>>> 
```